### DISCUSSION TITLE
Yoda and iRODS federation

### DISCUSSION THREAD

**Discussion by ll4strw** [UtrechtUniversity/yoda#342] [Q&A]
Hello yoda team, one of the most attractive features of iRODS is the possibility of federating multiple zones. Is this a feature that is also fully supported by yoda? For instance, given the right iRODS setup, would a yoda user of one zone be able to browse a federated zone? Are the iRODS zones of Utrecht University federated? Are you aware of any production implementation of a federated yoda cluster? Thanks in advance. Leonardo

**Comment by lwesterhof** [comment]
Hi Leonardo,

- currently federation is not fully supported, see issue https://github.com/UtrechtUniversity/yoda/issues/82
- our zones are not federated
- we are not aware of any production implementation of a federated Yoda cluster
